core/src/main/scala/io/finch/route/RouterN.scala

```diff
@@ -41,9 +41,14 @@ trait RouterN[+A] { self =>
   def apply(route: Route): Option[(Route, A)]
 
   /**
+   * Attempts to match a route, but only returns any unmatched elements, not the value.
+   */
+  def exec(route: Route): Option[Route] = apply(route).map(_._1)
```

Comment:
I remember you were talking about using `run` instead of `apply`? Do you think we can do it now? We still can have `apply` (marked as `@deprecated`) as an alias for `run`. Then `exec` might be renamed to `runIgnoreValue` (mark it as package-private method, since we only use it for tesging). What do you think?